src/MongoDB.Driver.Legacy/CommandResults/MapReduceResult.cs

```diff
@@ -84,6 +84,7 @@ public string DatabaseName
         /// <summary>
         /// Gets the duration.
         /// </summary>
+        [Obsolete("Duration is ignored by server versions 4.3.1 and newer.")]
```

Comment:
Some fields in the server response were removed. Then, the fields in this file cannot be parsed anymore.
Compare for details:
```
4.2.0
Response:
{
	"result": {
		"db": "Tests02282305",
		"collection": "Tests02282305.MapReduceOutputToCollectionOperationTestsOutput"
	},
	"timeMillis": 113,
	"counts": {
		"input": 2,
		"emit": 2,
		"reduce": 0,
		"output": 2
	},
	"ok": 1.0,
	"$clusterTime": {
		"clusterTime": Timestamp(1582920413, 4),
		"signature": {
			"hash": new BinData(0, "AAAAAAAAAAAAAAAAAAAAAAAAAAA="),
			"keyId": NumberLong(0)
		}
	},
	"operationTime": Timestamp(1582920413, 4)
}
4.3.4
Response:
{
	"result": {
		"db": "Tests02282310",
		"collection": "Tests02282310.MapReduceOutputToCollectionOperationTestsOutput"
	},
	"ok": 1.0,
	"$clusterTime": {
		"clusterTime": Timestamp(1582920701, 5),
		"signature": {
			"hash": new BinData(0, "AAAAAAAAAAAAAAAAAAAAAAAAAAA="),
			"keyId": NumberLong(0)
		}
	},
	"operationTime": Timestamp(1582920701, 5)
}
```

Reply:
Slight rewording requested:

    [Obsolete("Duration is not available on server versions 4.3.1 and newer.")]

Reply:
done